src/robot/libraries/BuiltIn.py

```diff
@@ -613,10 +613,18 @@ def should_be_equal(self, first, second, msg=None, values=True):
         for example, string ``false`` or ``no values``. See `Boolean arguments`
         section for more details.
 
+        If ignore_case is True, it indicates that ``first`` and ``second`` should be
```

Comment:
Should have double-backticks around `ignore_case` similarly as with other arguments. Also with the other keyword.